src/rpc/server.cpp

```diff
@@ -258,6 +258,22 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     return "Bitcoin server stopping";
 }
 
+UniValue uptime(const JSONRPCRequest& jsonRequest)
```

Comment:
> Maybe let's wait for other people's opinion on this?

certainly. Just my opinion that this should live in misc.cpp. I'm happy to defer to the consensus opinion.